planets.rb

```diff
@@ -0,0 +1,152 @@
+
+class SolarSystem
+  attr_accessor :planets, :ranked_planets
+
+  # Stores an array of hashes of info on planets
+  def initialize (planets)
+    @planets = planets
+    @ranked_planets = []
```

Comment:
Do you need 2 instance variables?  This looks like it should be a local variable holding an array of planet names (and numbers).